logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/MLSConversationsVerificationStatusesHandler.kt

```diff
@@ -45,12 +56,21 @@ internal interface MLSConversationsVerificationStatusesHandler {
     suspend operator fun invoke()
 }
 
+data class VerificationStatusData(
+    val conversationId: ConversationId,
+    val currentStatusInDatabase: VerificationStatus,
```

Comment:
This guy doesn't need to know how we are persisting data.
```suggestion
    val currentPersistedStatus: VerificationStatus,
```